cranelift-codegen/meta-python/isa/x86/encodings.py

```diff
@@ -341,6 +341,13 @@ def enc_i32_i64_ld_st(inst, w_bit, recipe, *args, **kwargs):
 X86_64.enc(base.copy_special, *r.copysp.rex(0x89, w=1))
 X86_32.enc(base.copy_special, *r.copysp(0x89))
 
+# Stack-slot-to-the-same-stack-slot copy, which is guaranteed to turn
+# into a no-op.
+X86_64.enc(base.copy_nop.i64, r.stacknull, 0)
+X86_64.enc(base.copy_nop.i32, r.stacknull, 0)
+X86_64.enc(base.copy_nop.f64, r.stacknull, 0)
+X86_64.enc(base.copy_nop.f32, r.stacknull, 0)
```

Comment:
There's something I've seen which might be equivalent to these 4 lines (worth a try at least!):

```
 X86_64.enc(base.copy_nop.any, r.stacknull, 0) 
```

Reply:
That produces
```
  File "/home/sewardj/MOZ/CL/may27/cranelift-codegen/meta-python/cdsl/instructions.py", line 341, in __getattr__
    assert name != 'any', 'Wildcard not allowed for ctrl_typevar'
```
so I backed it out.  I do notice other uses of `.any` in encodings.py, but they all have a type-descriptor fragment before the `.any`, for example `base.istore32.i64.any` and `base.jump_table_entry.i64.any.any`.